### DISCUSSION TITLE
rename/add suffix to flow runs

### DISCUSSION THREAD

**Discussion by i-sepulveda-reden-solar** [PrefectHQ/prefect#5433] [Q&A]
Hello,

I'd love to know if is possible to change or at least be ablo to rename/(add a suffix) the the flow runs created by the scheduler. they have random names for each schedule.

but imagine that you create 1 flow that has 2 executions everyday 1 is to get data from company A and the other is to get data from company B.

Will it be possible to add to each flow run name like : "-company_A" "-company_B"

I hope it is well explained.

thanks

**Comment by taljaards** [comment]
Hi @i-sepulveda-reden-solar 
You can look here for one option: https://discourse.prefect.io/t/how-can-i-set-a-custom-flow-run-name-instead-of-using-the-randomly-generated-adjective-noun-combination/131
